pycontrails/models/cocip/wake_vortex.py

```diff
@@ -1,4 +1,24 @@
-"""Wave-vortex downwash functions."""
+"""Wave-vortex downwash functions.
+
+This module includes both equations from the original CoCiP model
+:cite:`schumannContrailCirrusPrediction2012` as well as improvements from
+:cite:`unterstrasserPropertiesYoungContrails2016`.
```

Comment:
Could you qualify in what sense the U16 version is an improvement? (Better agreement with LES?)

Reply:
Parameterised ice crystal survival fraction based on large eddy simulations